### PR TITLE
add rubbish code that does nothing

### PR DESCRIPTION
**Pull request by ErikPDev**
notice: the typo is on purpose

### CODE REVIEW

File: src/Main.php [hunk 1]
```diff
@@ -3269,4 +5270,8 @@ public function onCommand(CommandSender $sender, Command $command, string $label
 		}
 		return true;
 	}
-}
\ No newline at end of file
+
+public function onDiesable(){
```
Comment (NgLam2911) [inline src/Main.php]: Please re-format this function.

**Review by NgLam2911 — DISMISSED** [review]
Everything is "ok", but need some changes...

File: src/Main.php (hunk 1 above)
Comment (ErikPDev) [reply src/Main.php]: wdym reformat?

Comment (NgLam2911) [reply src/Main.php]: hmm i will do it for u...

**Pull request merged**